index.html

```diff
@@ -4,9 +4,126 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>RoofTop-Cafe</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet'>
 </head>
 <body>
-    
+    <header>
+    <div class="nav-menu">
+        <i><img src="./images/menu-bar.jpg" alt="menu bars"></i>
+    </div>
+    <div class="mobile-menu">
+        <ul>
+            <li><a href="./index.html">Home</a></li>
+            <hr class="line-mobile-menu">
+            <li><a href="./about.html">About</a></li>
+            <hr class="line-mobile-menu">
+            <li><a href="#">Menu</a></li>
+            <hr class="line-mobile-menu">
+            <div class="closeMenu"><i><img src="images/cross-sign.png" alt="cross icone"></i></div>
+        </ul>
+    </div>
+
+    <div class="nav-black">
+        <div class="nav-icons">
+        <ion-icon name="logo-facebook" class="icon"></ion-icon>
+        <ion-icon name="logo-twitter" class="icon"></ion-icon>
+        <span class="tabs">English</span>
+        <span class="tabs">Language</span>
+    </div>
+      </div>
+
+        <div class="nav-desktop">
+            <div class="nav-logo">
+                <a href="./index.html"><img src="./images/logo.png" alt="Logo"></a>
+            </div>
+            <div class="nav-items">
```

Comment:
[OPTIONAL] Please use semantic nav element instead of div for wrapping your nav links to make your web page more SEO friendly.